local-development/developer-readme.md

```diff
@@ -0,0 +1,37 @@
+# OIBus developer handbook
+
+### Steps to try out the application:
+
+- Clone the repository
+- Run command `npm install` in project root
+- Run command `npm run build-client` (creates a build folder for the frontend bundle javascript files) 
+- Run command `npm run start` (this will start both the backend and frontend)
+- Open up in the browser the following url: `http://localhost:2223`. The port is specified in the `defaultConfig.json` file (currently 2223 is the default, it can be changed locally in your own config file generated at project startup)
+
+The project is up and running, but currently there are no south or north is specified.
+A simple way to try out the functionality is to create a `FolderScanner` south data source and a `Console` north application. The folder scanner scans the content of the given path (in the tests folder) periodically and sends the changes to the console.
+
+### Run database servers
+
+With the help of the docker-compose file, we can run a few databases by one command.
+- Open a terminal in the `local-deleopment` folder
+- Run command `docker-compose --env-file ./.env.dev up`
+The following services will start: **mysql, mssql, postgresql, mongodb, timescaledb, influxdb, rabbitmq, mosquitto**.
```

Comment:
Can you add an opcua server ? This one seems good with automatically generated data : https://github.com/Azure-Samples/iot-edge-opc-plc